ipapython/ipa_log_manager.py

```diff
@@ -54,6 +54,20 @@
 # Used by standard_logging_setup() for file message
 LOGGING_FORMAT_STANDARD_FILE = '%(asctime)s %(levelname)s %(message)s'
 
+# Used by standard_logging_setup() for command line output
+LOGGING_FORMAT_INFO_CMD = '%(message)s'
+
+
+# Additional info_log level methods
+def log_info_cmd(obj, msg, *args, **kwargs):
```

Comment:
Please use self to indicate it's going to be a method:

``def log_info_cmd(self, msg, *args, **kwargs)``

Reply:
This is used any more with the latest version.